pipelines/storage_network/Dockerfile.setup

```diff
@@ -25,10 +25,7 @@ RUN wget -q https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_V
     wget -q "https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64" && \
     mv yq_linux_amd64 /usr/local/bin/yq && \
     chmod +x /usr/local/bin/yq && \
-    apk add openssl openssh-client ca-certificates git rsync bash curl jq python3 py3-pip && \
-    ssh-keygen -t rsa -b 4096 -N "" -f ~/.ssh/id_rsa && \
-    curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 && \
-    chmod 700 get_helm.sh && \
-    ./get_helm.sh
+    apk add openssl openssh-client ca-certificates git rsync bash curl jq python3 py3-pip aws-cli && \
+    ssh-keygen -t rsa -b 4096 -N "" -f ~/.ssh/id_rsa
```

Comment:
kubectl version looks pretty old. update the versions of these dependencies if needed later.

Reply:
There is version information scattered throughout many files. We should think of how to centralize version control to reflect all changes in one place for all pipelines.

Reply:
There is a ticket for this: https://github.com/longhorn/longhorn/issues/6485.

I'll work on it later.

Reply:
Very good.